src/js/components/ErrorNotice.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react"
+
+import logInto from "../../../test/unit/helpers/loginTo"
+import provide from "../../../test/unit/helpers/provide"
+import Notice from "../state/Notice"
+import ErrorFactory from "../models/ErrorFactory"
+import ErrorNotice from "./ErrorNotice"
+import {screen} from "@testing-library/react"
+
+test("renders Error notice with no details", async () => {
+  const {store} = await logInto("workspace1", "pool1")
```

Comment:
These tests are great! However, I'd like to use the `setupBrim` helpers instead of the logInto and Provide helpers. 

There is an example usage in Preferences.test.tsx and in ReleaseNotes.test.tsx

The reason is because it exercises more of the same codepaths that the app will go through in production.

Reply:
Done, good call 👍